fvm/src/gas/price_list.rs

```diff
@@ -244,6 +357,17 @@ pub struct PriceList {
     pub(crate) verify_post_discount: bool,
     pub(crate) verify_consensus_fault: i64,
     pub(crate) verify_replica_update: i64,
+    // 1 Fuel = gas_per_fuel * 1 Gas
+    pub(crate) gas_per_fuel: i64,
+
+    pub(crate) extern_traversal_cost: i64,
+    // TODO: Some of these will probably need per-byte multipliers
+    pub(crate) block_open: i64,
+    pub(crate) block_read: i64,
+    pub(crate) block_create: i64,
+    pub(crate) block_link_base: i64,
+    pub(crate) block_link_per_byte: i64,
```

Comment:
This writes the blockstore from the block registry to the buffered blockstore (memory). It won't ultimately write to the disk blockstore until Machine flush, at which point we do copy_rec and only write it if it's reachable.